doc/lua_api.txt

```diff
@@ -2207,11 +2211,18 @@ These functions return the leftover itemstack.
 * `minetest.forceload_block(pos)`
     * forceloads the position `pos`.
     * returns `true` if area could be forceloaded
+    * Please note that forceloaded areas are saved when the server restarts.
 
 * `minetest.forceload_free_block(pos)`
     * stops forceloading the position `pos`
 
-Please note that forceloaded areas are saved when the server restarts.
+* `minetest.request_insecure_environment()`:
+  Returns an environment containing insecure functions if the calling mod has
+  been listed as trusted in the secure.trusted_mods setting, otherwise returns
```

Comment:
The doc should point out that `request_insecure_environment()` also returns an insecure environment when mod security has been disabled.